src/dbus_api.rs

```diff
@@ -241,6 +296,11 @@ fn create_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
             let dbus_context_clone = dbus_context.clone();
             let object_path: dbus::Path = create_dbus_pool(dbus_context_clone);
             let (rc, rs) = code_to_message_items(StratisErrorEnum::STRATIS_OK);
+
+            let dbus_context_clone = dbus_context.clone();
```

Comment:
What is the news on the cloning? I think at one time @agrover suggested that the context should be cloned within the ```create_dbus_pool()``` method.